changelog.json

```diff
@@ -0,0 +1,16 @@
+{
+  "package_name": "clnapp",
+  "version": "v0.0.1-alpha.2",
+  "api": {
+    "name": "github",
+    "repository": "dart-lightning/lndart.clnapp",
+    "branch": "main"
+  },
+  "generation_method": {
+    "name": "semver-v2",
+    "header_filter": true
```

Comment:
```suggestion
    "header_filter": false
```

This is used when you have somethings like multipackage, so you can filter by scope es. `fix(plugin)`

Reply:
ooh okay thanks